db/seeds.rb

```diff
@@ -43,7 +43,7 @@
     twitter_username: Faker::Internet.username(specifier: name),
     email_comment_notifications: false,
     email_follower_notifications: false,
-    email: Faker::Internet.email(name: name, separators: "+"),
+    email: Faker::Internet.email(name: name, separators: "+")[-49..-1], # Emails limited to 50 characters
```

Comment:
This returns nil:

```ruby
[13] pry(main)> Faker::Internet.email(name: Faker::Name.name, separators: "+")[-49..-1]
=> nil
```

What if instead you generate it from a short domain?

```ruby
name = "x" * 30
[19] pry(main)> name = "x"* 30
=> "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
[20] pry(main)> Faker::Internet.email(name: name)
=> "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx@weimannjaskolski.net"
[21] pry(main)> Faker::Internet.email(name: name, domain: Faker::Internet.domain_word.first(20))
=> "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx@cormier.co"
```

this way you're always sure it's less than 50